src/resizing.jl

```diff
@@ -104,19 +122,42 @@ function imresize(original::AbstractArray{T,N}, new_size::Dims{N}; kwargs...) wh
         if axes(dest) == inds
             copyto!(dest, original)
         else
+            # Non 1-based case as a fallback solution
+            # The OffsetArray case is specially handled to also output OffsetArray
             copyto!(dest, CartesianIndices(axes(dest)), original, CartesianIndices(inds))
         end
     else
         imresize!(similar(original, Tnew, new_size), original; kwargs...)
     end
 end
 
+function imresize(original::OffsetArray{T,N}, new_size::Dims{N}; kwargs...) where {T,N}
```

Comment:
OffsetArray is not necessarily the only array type that doesn't have indexing starting with 1; there are several other examples. What's the reason for the special method? It looks almost identical to the above.